catalog/justfile

```diff
@@ -82,11 +84,12 @@ shell:
     env DC_USER="airflow" just ../exec {{ SERVICE }} /bin/bash
 
 # Launch an IPython shell in a new container under `SERVICE`
+[positional-arguments]
 ipython *args: up-deps
     env DC_USER="airflow" just ../run \
         --workdir /opt/airflow/catalog/dags \
         {{ SERVICE }} \
-        bash -c \'ipython {{ args }}\'
+        bash -c "ipython ${@:2}"
```

Comment:
@sarayourfriend why was `${@:2}` used here? There doesn't appear to be a positional argument that we need to ignore, and it seems like the first argument after the command provided would be swallowed (for instance `ov just catalog/ipython  --TerminalInteractiveShell.editing_mode=vi` would no longer work as expected because the `editing_mode` argument would be ignored). Was that intentional?